webapp-django/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -119,6 +132,46 @@ def check_crashids(entropy_chunk, date):
     return list(set(missing))
 
 
+def check_crashids_for_date(firstchars_chunk, date):
+    """Check crash ids for a given firstchars and date"""
+    s3_context = get_s3_context()
+    bucket = s3_context.config.bucket_name
+    s3_client = s3_context.client
+
+    supersearch = SuperSearchUnredacted()
+
+    missing = []
+
+    for firstchars in firstchars_chunk:
```

Comment:
At some point, we might want to change how much chunkification happens. Maybe 2-characters or 1-character is better?